src/app/(main)/dashboard/page.tsx

```diff
@@ -1,16 +1,30 @@
+import { Suspense } from "react";
+
 import MyProgressContainer from "@/components/dashboard/my-progress/my-progress-container";
 import RecentlyTodoContainer from "@/components/dashboard/recently-todo/recently-todo-container";
 import TodosByGoalContainer from "@/components/dashboard/todos-by-goal/todos-by-goal-container";
+import getQueryClient from "@/lib/get-query-client";
+import {
+  getDashboardOptions,
+  getInfiniteGoalsOptions,
+} from "@/services/dashboard/query";
+
+export default async function DashboardPage() {
+  const queryClient = getQueryClient();
+
+  queryClient.prefetchQuery(getDashboardOptions());
+  queryClient.prefetchInfiniteQuery(getInfiniteGoalsOptions({}));
 
-export default function DashboardPage() {
   return (
-    <div className="grid h-full min-h-screen w-full grid-rows-[repeat(2,_258px)_1fr] gap-x-24 gap-y-8 overflow-y-scroll bg-slate-100 px-16 md:grid-cols-2 md:grid-rows-[auto_250px_1fr]">
+    <div className="grid w-full max-w-1203 grid-rows-[repeat(2,_258px)_1fr] gap-x-24 gap-y-8 bg-slate-100 px-16 pt-16 text-slate-800 md:grid-cols-2 md:grid-rows-[auto_250px_1fr]">
```

Comment:
데스크탑 화면에서 좌측 패딩이 디자인과 달라서 `lg:pl-77` 추가하면 좋을 것 같아요!
그리고 스크롤을 아래로 내렸을 때 하단에 약간의 패딩이 있는게 보기 더 좋을 것 같은데 pt -> py 변경하는 건 어떠실까요?

```suggestion
    <div className="grid w-full max-w-1203 grid-rows-[repeat(2,_258px)_1fr] gap-x-24 gap-y-8 bg-slate-100 px-16 py-16 text-slate-800 md:grid-cols-2 md:grid-rows-[auto_250px_1fr] md:py-24 lg:pl-77">
```

Reply:
figma 디자인에 맞는 스타일이네요 수정해서 올렸습니다.